nginx-agent-docker/container/start.sh

```diff
@@ -13,10 +13,11 @@ if [[ ! -z "$NIM_TAGS" ]]; then
    PARM="${PARM} --tags $NIM_TAGS"
 fi
 
+
 if [[ "$NAP_WAF" == "true" ]]; then
    PARM="${PARM} --nginx-app-protect-report-interval 15s --nap-monitoring-collector-buffer-size 50000 --nap-monitoring-processor-buffer-size 50000 --nap-monitoring-syslog-ip 127.0.0.1 --nap-monitoring-syslog-port 514"
-   /opt/app_protect/bin/bd_agent &
-   /usr/share/ts/bin/bd-socket-plugin tmm_count 4 proc_cpuinfo_cpu_mhz 2000000 total_xml_memory 471859200 total_umu_max_size 3129344 sys_max_account_id 1024 no_static_config &
+   su - nginx -s /bin/bash -c "/opt/app_protect/bin/bd_agent &"
```

Comment:
doesn't setting the `USER` in the Dockerfile work?

Reply:
Some processes must start as root and some with lower privileges, `USER` apparently doesn't make things work